code/modules/random_map/noise/ore.dm

```diff
@@ -124,4 +124,44 @@
 	deep_val = 0.8
 	rare_val = 0.7
 	min_rare_ratio = 0.02
-	min_rare_ratio = 0.01
\ No newline at end of file
+	min_rare_ratio = 0.01
+
+/datum/random_map/noise/ore/ice
+	surface_metals = list(
+		/decl/material/aspium_ice =		list(RESOURCE_MID_MIN, RESOURCE_MID_MAX),
+		/decl/material/lukrite_ice =	list(RESOURCE_MID_MIN, RESOURCE_MID_MAX),
+		/decl/material/rubenium_ice =	list(RESOURCE_MID_MIN, RESOURCE_MID_MAX),
+		/decl/material/trigarite_ice =	list(RESOURCE_MID_MIN, RESOURCE_MID_MAX),
+		/decl/material/ediroite_ice =	list(RESOURCE_MID_MIN, RESOURCE_MID_MAX),
+		MAT_IRON =              list(RESOURCE_MID_MIN,  RESOURCE_MID_MAX),
+		MAT_ALUMINIUM =         list(RESOURCE_LOW_MIN,  RESOURCE_LOW_MIN),
+		MAT_GOLD =              list(RESOURCE_LOW_MIN,  RESOURCE_LOW_MAX),
+		MAT_SILVER =            list(RESOURCE_LOW_MIN,  RESOURCE_LOW_MAX),
+		MAT_URANIUM =           list(RESOURCE_LOW_MIN,  RESOURCE_LOW_MAX)
+	)
+	rare_metals = list(
+		/decl/material/aspium_ice =		list(RESOURCE_MID_MIN, RESOURCE_MID_MAX),
+		/decl/material/lukrite_ice =	list(RESOURCE_MID_MIN, RESOURCE_MID_MAX),
+		/decl/material/rubenium_ice =	list(RESOURCE_MID_MIN, RESOURCE_MID_MAX),
+		/decl/material/trigarite_ice =	list(RESOURCE_MID_MIN, RESOURCE_MID_MAX),
+		/decl/material/ediroite_ice =	list(RESOURCE_MID_MIN, RESOURCE_MID_MAX),
+		MAT_GOLD =              list(RESOURCE_MID_MIN,  RESOURCE_MID_MAX),
+		MAT_SILVER =            list(RESOURCE_MID_MIN,  RESOURCE_MID_MAX),
+		MAT_URANIUM =           list(RESOURCE_MID_MIN,  RESOURCE_MID_MAX),
+		MAT_PHORON =            list(RESOURCE_MID_MIN,  RESOURCE_MID_MAX),
+		MAT_OSMIUM =            list(RESOURCE_MID_MIN,  RESOURCE_MID_MAX),
+		MAT_RUTILE =            list(RESOURCE_MID_MIN,  RESOURCE_MID_MAX)
+	)
+	deep_metals = list(
+		/decl/material/aspium_ice =		list(RESOURCE_MID_MIN, RESOURCE_MID_MAX),
```

Comment:
For consistency's sake please use a define for these paths. I am probably going to go through and remove them at some point but until then it should really not be mixed,